src/lenten-chili.md

```diff
@@ -0,0 +1,41 @@
+# Lenten (Vegan) Chili
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 30 min
+- 🍽️ Servings: 4
+
+## Ingredients
+
+- 1 large onion, chopped
+- 3 cloves garlic, minced
+- 4 15oz cans of kidney beans (or 6 cups cooked beans)
+- 1 29oz large can tomato sauce (or 4 cups fresh tomato sauce)
+- 1 15oz can diced tomatoes (or 1 large diced tomato with skin removed)
+- 1/2 can of sweet corn
+- 1 cup cooked white or brown rice
+- 1 tsp cumin
+- 1 tsp smoked paprika
+- 1 tsp garlic powder
+- 1 tsp onion powder
+- 1 tbsp chili powder
+- 1 tbsp crushed red pepper
+- 2-3 tbsp olive oil
+
+## Directions
+
+1. Chop the large onion and mince the garlic cloves.
+2. On medium-low heat in a large tall pot add 2-3 tbsp olive oil.
+3. Add the chopped onions into the pot with oil and constantly stir for 5-10 minutes until they begin to brown.
+4. Just before onions are done add the minced garlic to the pot and let cook for 1-2 minutes.
+5. Add the tomato sauce, diced tomatoes, sweet corn, beans, and rice.
+6. Turn up the heat to medium.
+6. Add each of the spices, taste the chili, and adjust flavoring accordingly.
+7. Let cook for 20-30 minutes, stirring occasionally.
+
+Let cool for a few minutes and then serve in bowls.
+
+## Contribution
+
+- George Z
+
+;tags: chili, vegan, lenten
```

Comment:
Tags should be written like so: `;tags: chili vegan lenten`, space seperated.

Reply:
fyi, the recipe checking script should catch this from now on